Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml

```diff
@@ -92,16 +95,32 @@
                                     <ColumnDefinition Width="60" />
                                     <ColumnDefinition />
                                 </Grid.ColumnDefinitions>
-                                
-                                <Image Grid.Column="0" 
-                                       Source="{Binding Path=Image}" />
+
+                                <Grid Grid.Column="0">
```

Comment:
The `MenuFlyout` doesn't have the property, and adding it to the `MenuFlyoutItem` which does have that property, doesn't seem to have any effect.